src/main/res/layout/progress_recycler_view.xml

```diff
@@ -15,8 +15,15 @@
         android:id="@+id/progress_recycler_view_progress_bar"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_centerVertical="true"
-        android:layout_centerHorizontal="true"
+        android:layout_gravity="center"
         />
 
-</RelativeLayout>
\ No newline at end of file
+    <FrameLayout
+        android:id="@+id/progress_recycler_view_empty_state_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
```

Comment:
I don't think we should make this assumption about what this layout is going to have... Users may not want to center their empty state here (I imagine a lot of use cases where they would want this, but who knows!).

I actually think we should have this width + height be `match_parent`, so we can let the users decide how this layout is.